### DISCUSSION TITLE
Action to merge changes made by script back into repo

### DISCUSSION THREAD

**Discussion by 1natalie** [leightonavantgardehaus/.github#17] [Q&A] [answered]
I currently have an action setup that will (1) checkout my repo (2) install node and dependencies (3) run a script that takes my directories/files and creates a JSON file with all their file paths. 

I am needing this action to take this JSON file and merge it back into my repo so that I may use the GitHub API to retrieve it.

**Comment by TL8N** [comment, marked as answer]
This action is probably the easiest way to achieve that: https://github.com/stefanzweifel/git-auto-commit-action
